data/ecosystem/vendors.yaml

```
@@ -222,6 +222,12 @@
   contact:
   oss: false
   commercial: true
+- name: MetricsHub
+  nativeOTLP: true
+  url: https://metricshub.com
+  contact:
```

```suggestion
  url: https://metricshub.com
  contact:
```

Please point to your documentation or a page that highlights that your product supports OTLP natively. Also provide a contact, ideally a github handle we can tag in case of change requests (can also be an email)

---

Thanks @svrnm for your feedback. Your comments have been taken into account.